src/client/components/map-box/map-box.vue

```diff
@@ -264,43 +322,50 @@ export default {
     }),
     selectFeature(id) {
       if (id) {
-          const feature = this.draw.get(id)
-          if (feature.geometry.type === 'Point') {
-            this.draw.changeMode('simple_select', { featureIds: [id] })
-            this.$emit('selectionchange', [feature])
-          } else {
-            this.draw.changeMode('direct_select', { featureId: id })
-          }
+        const feature = this.draw.get(id)
+        if (feature.geometry.type === 'Point') {
+          this.draw.changeMode('simple_select', { featureIds: [id] })
+          this.$emit('selectionchange', [feature])
+        } else {
+          this.draw.changeMode('direct_select', { featureId: id })
         }
+      }
     },
     clearMap() {
-      this.map.getLayer('projectArea-line') && this.map.removeLayer('projectArea-line')
-      this.map.getSource('projectArea-line') && this.map.removeSource('projectArea-line')
+      this.map.getLayer('projectArea-line') &&
+        this.map.removeLayer('projectArea-line')
+      this.map.getSource('projectArea-line') &&
+        this.map.removeSource('projectArea-line')
       this.draw.deleteAll()
       this.areas.forEach(area => {
-        this.map.getLayer(`${area.properties.name}-line`) && this.map.removeLayer(`${area.properties.name}-line`)
-        this.map.getSource(`${area.properties.name}-line`) && this.map.removeSource(`${area.properties.name}-line`)
+        this.map.getLayer(`${area.properties.name}-line`) &&
+          this.map.removeLayer(`${area.properties.name}-line`)
+        this.map.getSource(`${area.properties.name}-line`) &&
+          this.map.removeSource(`${area.properties.name}-line`)
```

Comment:
Please try to reset this indentation. spanning this over multiple lines makes it harder to read